src/typegen.ts

```diff
@@ -16,14 +17,31 @@ export function generate(options: Options): Promise<void> {
   return doGenerate(false, options)
 }
 
+function isModule(name: string) {
```

Comment:
```suggestion
function isPackage(name: string) {
```

This seems to leak in the face of e.g. `from 'fs-jetpack/foo`? I think you can go further in the guard by checking for leading `./`.